.github/workflows/snyk-scan.yml

```diff
@@ -20,8 +24,16 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: '3.10'
+      
+      - name: Check changed Deps files
+        uses: tj-actions/changed-files@v35
+        id: changed-files
+        with:
+          files: | # This will match all the files with below patterns
+            Pipfile
```

Comment:
what is the intention here?

Reply:
There might be scenario if some one make PR and although he does not add or remove any packages it still might fail due to previous issues. This is to avoid that.